bogie/process.go

```diff
@@ -90,10 +91,23 @@ func genContext(envfile string) (*context, error) {
 	return &c, nil
 }
 
+func fileExists(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
```

Comment:
Should be a blank line after this function.  Try running `gofmt` and see if it only adds the blank line - if it makes a ton of changes, then hold off and just add the blank line.

Reply:
Actually gofmt didn't add the empty line.   I did it manually.  It did update the imports order though.

Reply:
Weird I though that was a `gofmt` thing - my bad